packages/default/scss/input/_layout.scss

```diff
@@ -250,11 +250,20 @@
     }
 
     .k-form {
-        input:not(.k-checkbox):not(.k-radio),
-        .k-widget:not(.k-calendar):not(.k-switch) {
+        input:not(.k-checkbox):not(.k-radio) {
```

Comment:
This will make **all descendant inputs** block, including native checkboxes and radio buttons, as well as inputs in our components (`.k-input` and hidden inputs). That, in terms, may toggle visibility for them as a side effect, depending on the actual setup.

If the idea is to have our components, namely `k-textbox` and `k-textarea` be block, then let's make them only. Or rather, make them flex.